### DISCUSSION TITLE
Not Logging Any Enriched Information

### DISCUSSION THREAD

**Discussion by mglodack** [RehanSaeed/Serilog.Exceptions#8] [Q&A]
I followed your example that is on the README, but I am unable to get my exceptions printed in a pretty format.

I'm currently using this as my logger.

``` cs
using Serilog;
using Serilog.Exceptions;

ILogger logger = new LoggerConfiguration()
    .Enrich.WithExceptionDetails()
    .WriteTo.Sink(new RollingFileSink(
        @"C:\logs",
        new JsonFormatter(renderMessage: true))
    .CreateLogger();
```

When I call

``` cs
_logger.Error(exception, "Hello World);
```

It doesn't seem to print it pretty. Is there something I am missing?

**Comment by RehanSaeed** [comment]
You mean pretty JSON format. This is not a feature of Serilog.Exceptions but of the `JsonFormatter` and `RollingFileSink`. In my ReadMe, I just manually formatted the JSON for readability.

You probably don't want to do this anyway. You should not use the JSON format for human readability, you should use it to use something like Kibana. See http://rehansaeed.com/logging-with-serilog-exceptions/ for more info.
